### ISSUE TITLE
Do not disable zooming.

### ISSUE THREAD

**Issue by blancoj** [samvera/hyrax#203]
`<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">`  should be `<meta name="viewport" content="width=device-width, initial-scale=1.0">`, via http://a11yproject.com/posts/never-use-maximum-scale/

**Comment by mjgiarlo** [comment]
Comments on this, @projecthydra-labs/hyrax-ui-ux-advisors?
